tests/unit/package_managers/test_rpm.py

```diff
@@ -0,0 +1,30 @@
+from unittest import mock
+
+import pytest
+
+from cachi2.core.package_managers.rpm.redhat import RedhatRpmsLock
+
+
+class TestRedhatRpmsLock:
```

Comment:
I'm missing some negative test cases for the class' field validators. Since Arch isn't consumed directly I'd add one more test case to this test class that also checks whether validation in the Arch model works as expected.